packages/epictetus/epictetus.3.1.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+synopsis: "Elegant Printer of Insanely Complex Tables Expressing Trees with Uneven Shapes"
+maintainer: "Marc Chevalier <github@marc-chevalier.com>"
+authors: "Marc Chevalier <github@marc-chevalier.com>"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.6.3"}
+  "ounit2" {with-test & >= "2.0.8"}
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+run-test: [
+  ["dune" "runtest"]
```

Comment:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs]
```

Reply:
I'll report that right now in the source repo. I really don't understand dune well enough!